xray/services/scan.go

```diff
@@ -226,11 +226,11 @@ type ImpactPathNode struct {
 }
 
 type Cve struct {
-	Id           string `json:"cve,omitempty"`
-	CvssV2Score  string `json:"cvss_v2_score,omitempty"`
-	CvssV2Vector string `json:"cvss_v2_vector,omitempty"`
-	CvssV3Score  string `json:"cvss_v3_score,omitempty"`
-	CvssV3Vector string `json:"cvss_v3_vector,omitempty"`
+	Id           string  `json:"cve,omitempty"`
+	CvssV2Score  float64 `json:"cvss_v2_score,omitempty"`
+	CvssV2Vector string  `json:"cvss_v2_vector,omitempty"`
+	CvssV3Score  float64 `json:"cvss_v3_score,omitempty"`
+	CvssV3Vector string  `json:"cvss_v3_vector,omitempty"`
```

Comment:
I suggest we avoid modifying this struct, since it is used for other purposes, and this change seem to be breaking them. If needed, we can add a new struct in ```report.go```.

Reply:
The API response for CVSScores are actually numerics, not string types so I presumed this was an oversight?

Reply:
@eyalbe4 any update?

Reply:
Let's leave them as strings for now. If this causes issues for the new reporting functionality, please let me know, and we'll find a solution for this.
You change simply breaks jfrog-cli-core, which uses this as a dependency.